styles.css

```diff
@@ -296,3 +296,112 @@
 .project-button:hover {
   box-shadow: 1px 8px 16px rgba(125, 115, 252, 0.48);
 }
+
+.contact-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.contact-container h3 {
+  width: 327px;
+  height: 35px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #9390ff;
+  margin-bottom: 24px;
+  margin-top: 120px;
+}
+
+.contact-container p {
+  width: 327px;
+  height: 105px;
+  font-family: Inter, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  color: #f5f5ff;
+  margin-bottom: 40px;
+}
+
+.form-input-container {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.input {
+  width: 327px;
+  height: 48px;
+  background-color: #fff;
+  margin: 40px 24px;
+  padding: 15px 16px;
+}
+
+.client-message {
+  width: 327px;
+  height: 148px;
+  background-color: #fff;
+  font-family: Inter, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #81818c;
+  padding: 12px 12px;
+}
+
+.submit-message-button {
+  width: 100px;
+  height: 48px;
+  padding: 12px 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 24px;
+  letter-spacing: 3%;
+  text-align: center;
+  color: #edecff;
+  margin-top: 40px;
+  background-color: #7d73fc;
+  border: none;
+  outline: none;
+}
+
```

Comment:
- [ ] Please, consider, that you set for the button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template. You can find this information in Figma template
<img width="739" alt="Screen Shot 2022-01-12 at 02 07 35" src="https://user-images.githubusercontent.com/49064106/149029075-fc58238e-81ef-4d18-a2d8-20210af3f5e6.png">